src/main/scala/no/ndla/listingapi/controller/InternController.scala

```diff
@@ -37,11 +37,11 @@ trait InternController {
     }
 
     post("/newcover") {
-      writeService.newCover(extract[NewCover](request.body))
+      writeService.newCover(extract[NewCover](request.body), "NDLA import script")
```

Comment:
Hmm...disse endepunktene benyttes av et import-script ikke sant?
Tror @narhen har lagt inn støtte i scriptene våre for å autentisere seg med en import-klient. Da kunne vi faktisk ha brukt de credentialsene her også, i stedet for å hardkode det inn? Da kunne vi behandlet disse endepunktene, og hvilken bruker det var på akkurat samme måte som vi vil gjøre det i ListingController også.